rs-react-form/src/components/uncontrolled-from/ucf.tsx

```diff
@@ -0,0 +1,149 @@
+      gender: fd.get('gender') as 'male' | 'female' | 'other',
+      acceptTnC: fd.get('term') === 'on',
+      country: String(fd.get('country') ?? ''),
+      picture: undefined,
+    };
+
+    if (raw.age !== undefined && raw.age !== null) {
+      raw.age = Number(raw.age);
+    }
+
+    const file = (fd.get('picture') as File) ?? null;
+    try {
+      raw.picture = file && file.size ? await fileToBase64(file) : '';
+    } catch (err) {
+      if (err instanceof Error) {
+        setErrors((prev) => ({
+          ...prev,
+          picture: err.message || 'File error',
+        }));
+      }
+      return;
+    }
+
+    try {
+      const validated = await formSchema.validate(raw, { abortEarly: false });
+      dispatch(addForm({ source: 'UCF', data: validated }));
+      onSuccess();
+    } catch (err: unknown) {
+      const fieldErrors: Record<string, string> = {};
+
+      if (err instanceof yup.ValidationError) {
+        if (err.inner && err.inner.length) {
+          for (const e of err.inner) {
+            if (e.path && !(e.path in fieldErrors))
+              fieldErrors[e.path] = e.message;
+          }
+        } else if (err.path) {
+          fieldErrors[err.path] = err.message;
+        }
+      } else {
+        console.error(err);
+      }
+      setErrors(fieldErrors);
+    }
+  };
+  return (
+    <>
+      <h4>Uncontrolled Form</h4>
+      <form ref={formRef} onSubmit={handleSubmit} noValidate>
+        <div className="field">
+          <label htmlFor="name">Name</label>
+          <input id="name" name="name" data-autofocus />
+          <div className="err">{errors.name}</div>
+        </div>
+
+        <div className="field">
+          <label htmlFor="age">Age</label>
+          <input id="age" name="age" type="number" />
+          <div className="err">{errors.age}</div>
+        </div>
+
+        <div className="field">
+          <label htmlFor="email">Email</label>
+          <input id="email" name="email" type="email" />
+          <div className="err">{errors.email}</div>
+        </div>
+
+        <div className="field">
+          <label htmlFor="password">Password</label>
+          <input id="password" name="password" type="password" />
+          <div className="err">{errors.password}</div>
+        </div>
+
+        <div className="field">
+          <label htmlFor="confirmPassword">Confirm password</label>
+          <input id="confirmPassword" name="confirmPassword" type="password" />
+          <div className="err">{errors.confirmPassword}</div>
+        </div>
+
+        <fieldset className="field">
+          <legend>Gender</legend>
+          <label>
+            <input type="radio" name="gender" value="Male" /> Male
+          </label>
+          <label>
+            <input type="radio" name="gender" value="Female" /> Female
+          </label>
+          <label>
+            <input type="radio" name="gender" value="Other" /> Other
```

Comment:
```suggestion
            <input type="radio" name="gender" value="male" /> Male
          </label>
          <label>
            <input type="radio" name="gender" value="female" /> Female
          </label>
          <label>
            <input type="radio" name="gender" value="other" /> Other
```
вот так поменяй и заработает эта форма